File: TypeScript/arrays-and-hashing/contains-duplicate/containsDuplicate.ts
class Solution {

    hasDuplicate(nums: number[]) : boolean {
        for(let i=0;i<nums.length;i++){
            for(let j=i+1;j<nums.length;j++){
                if(nums[i]===nums[j])
                    return true;
            }
        }
        return false;
    }

}

const nums: number[] = [2,2,1,3];
const ifHasDuplicate:boolean = new Solution().hasDuplicate(nums);
console.log(ifHasDuplicate);

File: Python/arrays-and-hashing/contains-duplicate/contains_duplicate.py
from typing import List

class Solution:
    def has_duplicate(self, nums: List[int]) -> bool:
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if(nums[i]==nums[j]):
                    return True
        return False
    
nums = [2,2,4,5]
if_has_duplicate = Solution().has_duplicate(nums)
print(if_has_duplicate)